docs/getting-started/clients/mobile/ios/index.mdx

```diff
@@ -247,12 +251,88 @@ Next, we need to configure your Visual Studio environment for development.
 3. Open the mobile solution file (`bitwarden-mobile.sln`) in the root of your local mobile
    repository
 
-4. In the top bar, you should be able to select iOS > iPhoneSimulator > select your model and click
-   run (or your physical device if you set one up)
+4. In the top bar, you should be able to select App > Debug > select your model and click run (or
+   your physical device if you set one up)
 
 </TabItem>
 </Tabs>
 
+## Building
+
+There are currently a few problems on Visual Studio for Mac for building correctly the projects, so
+if you encounter some errors, build using the CLI being into the _src/App_ folder (previously
+removing bin/obj folders):
+
+For device:
+
+```
+dotnet build -f net8.0-ios -c Debug -r ios-arm64
+```
+
+For simulator:
+
+```
+dotnet build -f net8.0-ios -c Debug -r iossimulator-x64
+```
```

Comment:
Thanks for the feedback Patrick 🎉 
This is because of VS for Mac that won't be updated anymore. VS Code should be able to build the projects correctly.
And I was referring to any kind of weird error presented by VS for Mac, like ditto errors; which are fixed by doing what's explained there but yes I can put it in a note container.